languages/en/thingtalk.genie

```diff
@@ -321,13 +322,58 @@ $root = {
         | ('tell me' | 'give me' | 'show me' | 'present' | 'retrieve' | 'pull up') table:complete_table
         | ('hey almond' | '') ('search' | 'find' | 'i want' | 'i need') table:with_filtered_table
         | ('hey almond' | '') 'what are' table:with_filtered_table ('' | '?')
+        | ('hey almond' | '') 'what are' table:pvp_filtered_table ('' | '?')
+        | ('hey almond' | '') 'what are' table:avp_filtered_table ('' | '?')
+        | ('hey almond' | '') 'what are' table:npi_filtered_table ('' | '?')
+        | ('hey almond' | '') 'what are' table:npv_filtered_table ('' | '?')
+        | ('hey almond' | '') 'what are' table:apv_filtered_table ('' | '?')
     ) => C.makeProgram(new Ast.Statement.Command(table, [C.notifyAction()]));
     ('hey almond' | 'please' | '') ('list' | 'enumerate') table:with_filtered_table => {
         if (!table.schema.is_list)
             return null;
         return C.makeProgram(new Ast.Statement.Command(table, [C.notifyAction()]));
     };
 
+    // now => get => notify (who questions)
+    !turking (
+          ('hey almond' | '') ('who' | 'which person' | 'which one' | 'which guy')  ('has' | 'got' ) table:who_npp_filtered_table ('' | '?')
```

Comment:
Should we go crazy with synonyms of "person" here? "individual", "human", ... ? "dude"?
Maybe we should only have "person", and use PPDB (or similar synonym database, eg WordNet) to replace during augmentation?

(interestingly, whether "guy" and "dude" are gendered are a matter of opinion and culture, and their use varies across English-speaking countries, which makes me think "guy" is not a great fit in the general templates and might be better as augmentation, where uncertainty can be accounted for)

Also, "which one" is kind of awkward? It seems to imply an existing set to choose from, and would likely be confusing when there is context.
Speaking of which... are you planning to incorporate context in any form? That is, if the previous statement produced a list of people, should the question become a refinement?

Reply:
I removed "guy" and "which one", and added "individual". I agree that we should keep the more generic ones and use augmentation for others. 
No context. One problem at a time.